### ISSUE TITLE
The scale is far too big on Ubuntu 17.04 (Unity)

### ISSUE THREAD

**Issue by joedborg** [nexus511/gpd-ubuntu-packages#5]
I've installed Ubuntu 17.04.  Install etc. was fine, but once logged in, the scale is way too big.  I can't even get to the setting to turn it down.

**Comment by nexus511** [comment]
I label this as a enhancement as there is more important stuff to be resolved first. This does not mean that I will not work on this.

I guess that this setting can be changed by the user.

If anyone has a PR on this, please feel free to submit against the current development branch.

**Comment by Herst** [comment]
Using the tweak tool you can change the global scale to `1` (only integers are possible) and then increase the font scale instead (assuming that this is the scrolling method used here).

**Comment by nexus511** [comment]
I see this on my device as well. It looks like gnome3 on Ubuntu 17.04. does some fancy auto-scaling stuff. When properly rotated, things are scaled far too much.

For now, I do not know how to disable this properly. On Ubuntu 16.04.3 there is no such problem at all.

**Comment by joedborg** [comment]
@Herst doesn't appear to be that unfortunetly.  Scale factor is 1 and if I turn that down (<1) only some of the desktop is scaled down

**Comment by nexus511** [comment]
@joedborg Yes, I can second that. I think that gnome3 somehow determines the DPI of the display which is actually quite high.

    xrandr --output DSI1 --scale 2x2

should help to temporarily bring up usable scaling, but I do not have a clue yet, how to solve this persistently.

**Comment by joedborg** [comment]
@nexus511 I think it's unity and gnome, right?

**Comment by nexus511** [comment]
@joedborg No idea. I am not an Ubuntu user. Personally I only use Xubuntu with XFCE4 or AwesomeWM.

I see that Ubuntu (as in Unity) detects the display properly as 7'' sized and I would guess that it decides then to just scale it up to match it's DPI.

As the desktop is also ignoring xorg.conf entries, there is not much space for any solutions that will properly work and can be deployed via packages. This looks pretty bad and I guess will require some time digging into it.

**Comment by omgmog** [comment]
I found the same problem with a clean install of Ubuntu 17.04. 

This is due to the dconf key `/com/ubuntu/user-interface/scale-factor ` being set to 32 (which is 4x scale)

The way the values work is that the scale is the value divided by 8:

```
10: 10/8 = 1.25x
12: 12/8 = 1.5x
16: 16/8 = 2x
24: 24/8 = 3x
32: 32/8 = 4x
```

Open a terminal (ctrl+alt+T), run the following:

```
dconf write /com/ubuntu/user-interface/scale-factor "{'DSI1': 12}"
```

In this case `DSI1` is the display in the GPD Pocket.

The scaling change should happen immediately, but if it doesn't you can reboot and it should change.

**Comment by nexus511** [comment]
@omgmog Great! Thank you!

That brings up the question, if it is possible to provide the value as a default. I try to look into that tomorrow.

**Comment by cawilliamson** [comment]
@nexus511 My ansible playbooks setup some dconf defaults in the xorg folder - may be worth taking a look if that's what you're looking to achieve. 👍

**Comment by nexus511** [comment]
I merged a patch from @sagebind to *development/India*. I will make this available with the next release.
